survey/survey_app/serializers.py

```diff
@@ -12,14 +13,16 @@ class Meta:
 class QuestionSerializer(serializers.ModelSerializer):
     options = OptionSerializer(many=True, read_only=True)
     text = serializers.CharField(required=False)
+    # Add the following line to make the 'survey' field required
+    survey = serializers.PrimaryKeyRelatedField(queryset=Survey.objects.all(), write_only=True)
 
     class Meta:
         model = Question
         fields = '__all__'
```

Comment:
We should not use `__all__` in this and should explicitly specify the fields we want to returns because of security and performance.

Reply:
class QuestionSerializer(serializers.ModelSerializer):
    options = OptionSerializer(many=True, read_only=True)
    survey = serializers.PrimaryKeyRelatedField(queryset=Survey.objects.all(), write_only=True)

    class Meta:
        model = Question
        fields = ['id', 'text', 'survey', 'options']
        extra_kwargs = {'text': {'required': False}}